site/documentation/static/css/hono.css

```diff
@@ -89,3 +89,11 @@
 .swagger-ui table, .swagger-ui thead, .swagger-ui td {
 	border: none;
 }
+
+/* prevents that the anchor is positioned behind the menu bar */
+:target:before {
+  content: "";
+  display: block;
+  height: 60px;
```

Comment:
is this number of pixels determined from the currently used heading font?

Reply:
No, this does not depend on the size of headings, but on that of the menu bar. The height of the menu bar is nowhere absolutely defined, so this is an approximation. The menu bar is usually 48px high (but this is calculated dynamically in the JS code for the sticky menu bar and could change if you put large elements into it).
TBH my solution is a bit hacky. I spent a lot of time to solve this for the theme used for the homepage part and did not find a better solution. So I am glad that this could easily be adapted for the documentation part.

Reply:
I see. Fair enough.